packages/SwingSet/src/controller.js

```diff
@@ -226,16 +225,10 @@ function buildSESKernel(sesEvaluator, endowments) {
   return buildKernel(endowments);
 }
 
-function buildNonSESKernel(endowments) {
-  // Evaluate shims to produce desired globals.
-  const evaluateOptions = makeDefaultEvaluateOptions();
-  // eslint-disable-next-line no-eval
-  (evaluateOptions.shims || []).forEach(shim => (1, eval)(shim));
-
-  return buildKernelNonSES(endowments);
-}
-
 export async function buildVatController(config, withSES = true, argv = []) {
+  if (!withSES) {
```

Comment:
Having a parameter in the middle of the list that's a mandatory `true` seems kinda icky.  Is it too late to fix the API to just get rid of it?

Reply:
Actually I'd say it's too early :). Once this is all done, I'm planning to go through the whole monorepo and find all the calls to `buildVatController` and remove that parameter from all of them (probably turning it into an options bag, or merging it all into the config object). For now, I wanted to minimize changes to downstream users.

Reply:
Yah, I figured as much.  BTW, outside `SwingSet/test` I count 9 calls to `buildVatController`, so we're not looking at a big job.